Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/Definitions/Result.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Frends.FixedWidthFlatFile.Parse.Definitions;
+
+public class Result
+{
+    public List<Dictionary<string, dynamic>> Data { get; internal set; }
```

Comment:
Could it be possible to have some predefined type for the value instead of dynamic? Like changing the data to string before adding to the dictionary? Downside of dynamic is that the behavior is unpredictable and users might have trouble understanding what's the type of that value if they processing that data.

Reply:
The best we can do is have an "object" instead of dynamic really, since the type of the data is defined in column specs, so data can be string/int/double/date/etc.

Reply:
And when I think of it - I do not know what would be a better way of dealing with this, since object is not a nice way of doing it either - it will not allow to call any methods on it without throwing an error really. Maybe wrapping this into some helper class with helpers like .ValueAsString and .ValueAsInt would be the only typed alternative, but in there to use the thing you would need to make sure that you are using the value correctly and calling the correct helper.

Reply:
Any suggestions are appreciated, but I think if we just return everything as string then it forces the users to make the casts and parses themselves, which is no better that actually just using this task in its current form.

Reply:
Ah, didn't notice that the dictionary has different value types. In this case, I don't think there is another way which would be better that using dynamic.